experimenter/tests/integration/nimbus/pages/demo_app/frontend.py

```diff
@@ -0,0 +1,47 @@
+import time
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from nimbus.pages.base import Base
+
+
+class DemoAppPage(Base):
+    def __init__(self, driver):
```

Comment:
Do we want this as ```DemoAppPage``` or would ```CirrusPage``` make more sense?

Reply:
Since this is about launching a demo app using the frontend part of that to make cirrus requests, that's why I named as Demo app to signify that we are standing Demo app